src/func_impl.hpp

```diff
@@ -59,33 +71,20 @@ class FuncImpl : public Func
     void return_(const IReg& retval);
     void return_();
 
-    const Syntfunc& getData() const { return m_data; }
+    const Syntfunc& getData() const { return m_subintervals; }
     
     int m_cmpopcode; // TODO(ch): IMPORTANT(CMPLCOND) delete this trivial workaround ASAP;
 private:
-    struct cflowbracket
-    {
-        enum { DO, DOIF, IF, ELSE};
-        size_t tag;
-        size_t labelOrPos;
-        size_t elifRepeats;
-        std::vector<size_t> breaks;
-        std::vector<size_t> continues;
-        cflowbracket(size_t a_tag, size_t a_labelOrPos, size_t a_elifRep = 0) : tag(a_tag), labelOrPos(a_labelOrPos), elifRepeats(a_elifRep) {}
-    };
-    std::deque<cflowbracket> m_cflowStack;
-    Syntfunc m_data;
+    std::deque<ControlFlowBracket> m_cflowStack;
+    Syntfunc m_subintervals;
     ContextImpl* m_context;
     size_t m_nextIdx;
     size_t m_nextLabelIdx;
-    size_t m_epilogueSize;
 
     enum {RT_NOTDEFINED, RT_REGISTER, RT_VOID};
     int m_returnType;
 
-    void allocateRegisters();
     void jumpificate();
```

Comment:
please, rename "jumpificate()" to something else

Reply:
Done. New name is controlBlocks2Jumps.